R/save_with_logo.R

```diff
@@ -80,7 +81,8 @@ save_with_logo <-
            width = 8,
            height = 8,
            dpi = 300,
-           suffix = NULL) {
+           suffix = NULL,
```

Comment:
The examples still contain the old colour palettes. I've updated them to Expl_RGB